src/seedu/addressbook/AddressBook.java

```diff
@@ -364,24 +387,27 @@ private static void loadDataFromStorage() {
      * @param userInputString  raw input from user
      * @return  feedback about how the command was executed
      */
+    public enum commandType {
+    	ADD, FIND, LIST, DELETE, CLEAR, HELP, EXIT
+    }
```

Comment:
Using enum for commandtype is a good idea! However, the implementation can be improved. Also do look at the java coding standard doc on how enum variables should be named.